Adaptors/S3/src/ObjectStorage.cs

```diff
@@ -213,76 +179,70 @@ await keyList.Chunk(MaxAllowedKeysPerDelete)
                        bucketName_);
```

Comment:
```suggestion
                       bucketName_);
         return false;
```

Actually, I'm not sure if you should return instead of rethrow.